shell.nix

```diff
@@ -1,4 +1,6 @@
-{ pkgs ? import <nixpkgs> { } }:
```

Comment:
Good catch

Reply:
I have no channels configured on my system, so these tend to fail pretty reliably. :)